packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -135,6 +136,24 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                         ],
                     },
                 },
+                ...(isQueryingAicsFms
```

Comment:
Did you create a ticket for adding this capability for non-aics data sources? I think it wouldn't be very difficult to do via SQL like:
```
UPDATE ${datasourceName}
SET ${columnToUpdate} = ${valueToReplaceWith}
WHERE ${here either select by row number or the internally made BFF ID} = ${list of row numbers or bff ids}
```

Reply:
honestly I forgot we were planning to allow editing on non-aics data sources. Will make a ticket!